ideas42ventures/ideas42ventures.com
# Integration of the ButtonDown API

Works on #89

- Removes references to Netlify forms
- Adds newsletter-submission function
- Adds ButtonDown API to handle email submissions

How to Test 🧪
1. Go to [link](https://deploy-preview-90--ideas42ventures.netlify.app/)
2. Submit an email address. 
- If it's **VALID** you should receive a success message.  
- If it's **INVALID** you should recieve an error message.
